File: templates/js/benchmarks/steady-state.js
// %imports

if(!performance){
	performance = {};
}else if(!performance.now){
    performance.now = Date.now;
}


async function runner(...args){
    let output = 0;
    // %init
	// %gc
    for(let i = 0;i<10;i++){
        // %pre
        // %content
        // %post
        // %gc
    }

    // %pre
    let start = performance.now();
    // %content
    let diff = performance.now();
    // %post
    // %gc
    let result = {
        input:JSON.stringify((args)),
        output:output,
        time:(diff - start)/1000
    };
    console.log(JSON.stringify(result));

}

module.exports = {runner};

// %runner_call

File: templates/py/benchmarks/steady-state.py
import numpy as np
import time
import json
import sys

def runner(*args):
    output=0
    # %vars
   
    for i in range(10):
        3
        # %pre
        # %content
        # %post
    # %pre
    start = time.time()
    # %content
    end = time.time()
    # %post
    res = {
        "input": args,
        "output": output,
        "time": end - start
    }
    json.dump(res, sys.stdout)


